src/test/java/redis/clients/jedis/tests/commands/AccessControlListCommandsTest.java

```diff
@@ -315,6 +316,84 @@ public void aclCatTest() {
     }
   }
 
+  @Test
+  public void aclLogTest() {
+    jedis.aclLog("RESET");
+    assertNotNull(jedis.aclLog()); // after reset the log should be null
```

Comment:
In comment, the world *null* may be confusing. Perhaps, *cleared* or something else?

Reply:
Removed and changed the test to `isEmpty()`